### ISSUE TITLE
have to manually shut down chromium to run normally

### ISSUE THREAD

**Issue by sir-reynolds** [chrome-php/headless-chromium-php#214]
Windows 10 / 64 bit
PHP 7.2
Chromium Version 90.0.4427.0 (Developer Build) (64-bit)

```
require_once __DIR__ . "/vendor/autoload.php";

use HeadlessChromium\BrowserFactory;

$browserFactory = new BrowserFactory( __DIR__ ."/browser/chrome-win/chrome.exe" );

/**
 * At this moment, browser will open but never navigate to url or never run anything.
 * Browser will idle , this like some bugs, because at this moment debug logger not starting.
 * i mark this jobs as "jobs A"
**/
$browser = $browserFactory->createBrowser([
	'windowSize'                    => 	[860, 680],
	'headless'		        => 	false,
	'ignoreCertificateErrors'	=>      true,
	'userAgent'		        =>	'Mozilla/5.0 (compatible; MSIE 10.0; Windows Phone 8.0; Trident/6.0; IEMobile/10.0; ARM; Touch; NOKIA; Lumia 920)',
	'debugLogger'                   =>      'php://stdout'
]);

$page = $browser->createPage();

$page->navigate( 'https://example.com' )->waitForNavigation();

$browser->close();
```

as I mentioned, there is a browser that is open but does nothing, and also until now the debuglogger is not running or showing nothing.

but when I close the browser that didn't do anything earlier, without waiting for a long time a new browser will appear and everything works normally.

everything is normal, the debuglogger displays information, the browser starts to open the url I want.

my question is, why do I have to close the first browser so that the second browser appears and runs normally.

I've done everything possible, tried local chrome and others. but everything remains the same, I have to close the first browser so that the second browser appears and runs normally.

is there an error that i don't know about? or does this framework have bugs

**Comment by guttume** [comment]
@sir-reynolds This is happening only when `debugLogger` option is passed while creating the browser. This is issue is specific to Windows OS.

After investigating the internal code, I found out that whenever the `debugLogger` option is passed, it tries to get the version detail of the chrome by running the chrome binary with `--version` flag. Accoding to this [reported bug](https://bugs.chromium.org/p/chromium/issues/detail?id=158372) at chromium, `--version` flag is not supported on Windows.

**Comment by GrahamCampbell** [comment]
Fixed in v0.10.0.
